### PR TITLE
P0 badge button

### PR DESCRIPTION
**Pull request by Muzammil-khan710**
Created Button and Badge components

### CODE REVIEW

File: docs/components/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+.bdg-med {
+    height: 15px;
+    width: 15px;
+    top: 3.5rem;
+    left: 0rem;
+    border: 3px solid white;
+}
+
+.bdg-lg {
+    height: 20px;
```
Comment (itsmebipul00) [inline docs/components/badge/badge.css]: Try using rems

**Review by itsmebipul00 — APPROVED** [review]

File: docs/components/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+    position: absolute;
+    border-radius: 50%;
+}
+
+.bdg-sm {
+    height: 12px;
+    width: 12px;
+    top: 2rem;
+    left: 0rem;
+    border: 1px solid white;
```
Comment (zoef-yt) [inline docs/components/badge/badge.css]: use var for colour

File: docs/components/badge/badge.css [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.bdg-med {
+    height: 15px;
+    width: 15px;
+    top: 3.5rem;
+    left: 0rem;
+    border: 3px solid white;
```
Comment (zoef-yt) [inline docs/components/badge/badge.css]: use var for colour

File: docs/components/badge/badge.css [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+}
+
+.bdg-lg {
+    height: 1.2rem;
+    width: 1.2rem;
+    top: 5rem;
+    left: 0rem;
+    border: 3px solid white;
```
Comment (zoef-yt) [inline docs/components/badge/badge.css]: use var for colour

**Review by zoef-yt — CHANGES_REQUESTED** [review]

**Review by zoef-yt — APPROVED** [review]
Good to merge.

**Pull request merged**